PurchasesTests/Networking/Backend/BackendGetCustomerInfoTests.swift

```diff
@@ -117,7 +117,7 @@ class BackendGetCustomerInfoTests: BaseBackendTests {
         let underlyingError = (error?.userInfo[NSUnderlyingErrorKey]) as? NSError
         expect(underlyingError).toEventuallyNot(beNil())
         expect(underlyingError?.domain).to(equal("RevenueCat.BackendErrorCode"))
-        expect(error?.userInfo["finishable"]).to(be(true))
+        expect(error?.userInfo["finishable"] as? Bool) == true
```

Comment:
out of curiosity, does `beTrue()` work?

Reply:
Yeah let me try that for this, should be simpler.

Reply:
Nope that would also require the cast:
> cannot convert value of type 'Any?' to expected argument type '() throws -> Bool?'